mkdocs.yml

```diff
@@ -267,6 +267,8 @@ nav:
           - 'Hardware Configuration Matrix': variants/protectli_vp46xx/hardware-matrix.md
           - 'Test matrix': variants/protectli_vp46xx/test-matrix.md
           - 'Openness score': variants/protectli_vp46xx/openness_score.md
+      - 'Odroid H4':
```

Comment:
Assembly guides are in one place now, see `Guides` around line 352

Reply:
file moved to mentioned guides

Reply:
https://github.com/Dasharo/docs/pull/878/commits/ee0640e340048aef8480a7013fb064660e0d7a6b